webapp-django/crashstats/crashstats/tests/test_archivescraper.py

```diff
@@ -227,6 +204,12 @@ def test_scrape_candidates(self, config, req_mock, db_conn):
         )
         req_mock.get(HOST + '/pub/firefox/releases/', text=release_page)
 
+        # Fill out the top-level urls with blank documents
+        req_mock.get(HOST + '/pub/devedition/releases/', text='<html></html>')
+        req_mock.get(HOST + '/pub/devedition/candidates/', text='<html></html>')
+        req_mock.get(HOST + '/pub/mobile/releases/', text='<html></html>')
+        req_mock.get(HOST + '/pub/mobile/candidates/', text='<html></html>')
```

Comment:
I changed from running a specific method to running the entire command. Because I did that, it looks at all the products and so I needed to add empty pages for them.